metrics/classification.go

```diff
@@ -37,6 +38,18 @@ func (cm *ConfusionMatrix) Update(yText, yHatText string) {
 	if err != nil {
 		log.Fatal("Error parsing int", err)
 	}
+
+	var yWeight int64
+	invalidWeight := strings.Contains(yWeightText, "|")
+	if !invalidWeight {
+		yWeight, err = strconv.ParseInt(yWeightText, 10, 64)
+		if err != nil {
+			log.Fatal("Error parsing int", err)
+		}
+	} else {
+		yWeight = 1
```

Comment:
If you change line 42 to be `yWeight := 1` then you can get rid of this else clause. What do you think?